docs/getting_started/basics/authoring.md

```diff
@@ -1,36 +1,44 @@
 # Workflow Authoring
 
-By opening Shift you will be prompted to an empty board.  
+This page is dedicated to explain the most basic steps for creating a Shift workflow. 
+When launching Shift, an new and empty board will be opened where nodes and connections can be added to start building the workflow. 
 
-## Creating Your First Node
+## Creating a Node
 
-To create a new [operator](../intro/terminology.md/#operator), you can either press the `Tab Key` or `Double Left-Click` with your mouse.  
+To create a new [node](../terminology.md/#operator), press the `Tab` key or `Double Left-Click` to open the "*Create Node*" dialog. This dialog enables searching for nodes by name and/or filtering by [catalog](../terminology.md/#catalog). The nodes will be listed alphabetically underneath, where they can be added to the Shift board by double-clicking on them.
 
-![Create Node Dialog](../images/node_creation_dialog.png "Create Node Dialog")  
+![Create Node Dialog](../../images/create_node_dialog.gif "Create Node Dialog")   
 
-Then you can choose the type of operator you want to create and double-click on its name to create it.
+Alternatively, nodes can be added to the board using the [Nodes List widget](../ui_overview.md/#operator) present in the Shift interface.
 
 ## Creating Connections
 
-If you want a certain operator to pass information to another one, you can connect their [plugs](../intro/terminology.md/#plug). To do that, you just need to `Left-Click and Drag` from one plug to the other plug you want to connect to. This will propagate whatever data is produced by the first operator and pass it to the connected plug.  
+To establish a connection between plugs from different nodes, `Left-Click` and `Drag` from one plug to the desired plug. By creating a connection, the workflow can propagate the data produced in the output plugs of a node and transfer it to the connected input plug of another node.
```

Comment:
Shouldn't we also put a link to the terminology for `plugs`?

Reply:
Done